.travis.yml

```diff
@@ -0,0 +1,42 @@
+language: cpp
+dist: trusty
+matrix:
+  include:
+  - os: linux
+    env: CONDA=3.6 RDKIT=2019.09.2
+  - os: linux
+    env: CONDA=3.7 RDKIT=2019.09.2
+  - os: linux
+    env: CONDA=3.8 RDKIT=2019.09.2
+  - os: osx
+    env: CONDA=3.6 RDKIT=2019.09.2
+  - os: osx
+    env: CONDA=3.7 RDKIT=2019.09.2
+  - os: osx
+    env: CONDA=3.8 RDKIT=2019.09.2
+
+before_install:
+- |
+  if [ "$TRAVIS_OS_NAME" = "linux" ]; then export CXX=g++-4.8 CC=gcc-4.8; fi
+  if [ -n "$PYTHON" ]; then
```

Comment:
Seems like there's no point in including this branch. We have an RDKit dependency and I really don't recommend using homebrew with the rdkit. How about just sticking to conda?

Reply:
I used a template that I should have cleaned a bit...